pal/inc/pal_mstypes.h

```diff
@@ -42,22 +42,11 @@ extern "C" {
 
 #ifdef __i386__
 
-#if !defined(__stdcall)
-#define __stdcall      __attribute__((stdcall))
-#endif
-#if !defined(_stdcall)
-#define _stdcall       __stdcall
-#endif
-
-#if !defined(__cdecl)
-#define __cdecl        __attribute__((cdecl))
-#endif
-#if !defined(_cdecl)
-#define _cdecl         __cdecl
-#endif
-#if !defined(CDECL)
-#define CDECL          __cdecl
-#endif
+#define __stdcall
```

Comment:
we already disabled it for x64. It is not because it doesn't support that. It is because of our usage of templates.